documentation/modules/virt-migration-workflow.adoc

```diff
@@ -30,10 +32,50 @@ The workflow describes the following steps:
 +
 *After the VM disks are transferred:*
 
-. The `Migration Controller` service creates a `conversion` pod with the PVCs attached to it.
+. The `Migration Controller` service creates a `conversion` pod with the PVCs attached to it when importing from VMWare.
 +
 The `conversion` pod runs `virt-v2v`, which installs and configures device drivers on the PVCs of the target VM.
++
+. The `Migration Controller` service creates a `VirtualMachine` CR for each source virtual machine (VM), connected to the PVCs.
+
+. If the VM was running on the source environment, the `Migration Controller` powers on the VM, the `KubeVirt Controller` service creates a `virt-launcher` pod and a `VirtualMachineInstance` CR.
++
+The `virt-launcher` pod runs `QEMU-KVM` with the PVCs attached as VM disks.
+
+*Cold migration (oVirt/OpenStack):*
 
-. When the target VM is powered on, the `KubeVirt Controller` service creates a `virt-launcher` pod and a `VirtualMachineInstance` CR.
+. When you create a `Migration` custom resource (CR) to run a migration plan, the `Migration Controller` service creates a `PersistentVolumeClaim` CR and a `OvirtVolumePopulator`/`OpenstackVolumePopulator` CR for each source VM disk.
++
+*For each VM disk:*
+
+. The `Populator Controller` service creates a temporarily persistent volume claim (PVC).
+. If the `StorageClass` has a dynamic provisioner, the persistent volume (PV) is dynamically provisioned by the `StorageClass` provisioner.
+. The `Migration Controller` service creates a dummy pod to bind the PVC, the pod contains `pvcinit` in his name.
+. The `Populator Controller` service creates a `populator` pod.
+. The `populator` pod transfers the disk data to the PV.
++
+*After the VM disks are transferred:*
+. The temporarily PVC is deleted, and the initial PVC will point to the PV with the data.
+. The `Migration Controller` service creates a `VirtualMachine` CR for each source virtual machine (VM), connected to the PVCs.
+. If the VM was running on the source environment, the `Migration Controller` powers on the VM, the `KubeVirt Controller` service creates a `virt-launcher` pod and a `VirtualMachineInstance` CR.
 +
 The `virt-launcher` pod runs `QEMU-KVM` with the PVCs attached as VM disks.
+
+*Cold migration (VMWare):*
+
+. When you create a `Migration` custom resource (CR) to run a migration plan, the `Migration Controller` service creates a `DataVolume` CR for each source VM disk.
++
+*For each VM disk:*
+
+. The `Containerized Data Importer (CDI) Controller` service creates a blank persistent volume claim (PVC) based on the parameters specified in the `DataVolume` CR.  
+. If the `StorageClass` has a dynamic provisioner, the persistent volume (PV) is dynamically provisioned by the `StorageClass` provisioner.
+. The `Migration Controller` service creates a dummy pod to bind the PVC, the pod contains `pvcinit` in his name.
+. The `Migration Controller` service creates a `conversion` pod.
```

Comment:
Ths does not really fit into the "For each VM disk" section. Same for the `pvcinit` pod (here and elsewhere). Although I am not sure if it's worth separating to a new section.

Reply:
i tried to pull it out (easily in vmware hard in ovirt because the flow). making it without separating to a new section will make the section much harder to read in my opinion

Reply:
Ok, let's leave it as it is now.